.github/CODEOWNERS

```diff
@@ -1,28 +1,28 @@
 # This does not cover all the code in edx-platform but it's a good start.
 
 # Core
-common/djangoapps/student/                 @edx/platform-core
-common/djangoapps/third_party_auth/        @edx/platform-authn
-common/lib/xmodule/xmodule/                @edx/platform-core
-lms/djangoapps/course_api/blocks           @edx/platform-core
-lms/djangoapps/courseware/                 @edx/platform-core
-lms/djangoapps/grades/                     @edx/platform-grades
-lms/djangoapps/instructor/                 @edx/platform-core
-lms/djangoapps/instructor_task/            @edx/platform-core
-lms/djangoapps/mobile_api/                 @edx/platform-mobile
-openedx/core/djangoapps/contentserver/     @edx/platform-core
-openedx/core/djangoapps/heartbeat/         @edx/platform-core
-openedx/core/djangoapps/oauth_dispatch     @edx/platform-authn
-openedx/core/djangoapps/user_api/          @edx/platform-authn
-openedx/core/djangoapps/user_authn/        @edx/platform-authn
-openedx/features/course_experience/        @edx/platform-courseware
+common/djangoapps/student/
```

Comment:
What does a line represent without a mapping?

Reply:
I wanted to leave the lines in for now because the grouping of core/extensions is  still useful to have.  It is the same as if the line were deleted.